### ISSUE TITLE
Port gcdmaster to Gnome 3

### ISSUE THREAD

**Issue by domichel** [cdrdao/cdrdao#3]
In my gentoo system, configuring cdrdao give:

```
checking for sigc++-2.0 >= 2.0.0... yes
checking for gtkmm-2.4 >= 2.4.0... yes
checking for libgnomeuimm-2.6 >= 2.6.0... no
<snip>
  Building gcdmaster : no
```
dev-cpp/libgnomeuimm disappeared from gentoo several years ago, and I guess from other distributions as well. It is provided by gnome-base/libgnomeui (2.24.5 installed) and I don't even know if it is a trivial fix for that issue.

```
cat /usr/lib64/pkgconfig/libgnomeui-2.0.pc
prefix=/usr
exec_prefix=${prefix}
libdir=/usr/lib64
includedir=${prefix}/include
libgnomeui_serverdir=${exec_prefix}/libexec

Name: libgnomeui
Description: libgnomeui
Requires: libgnome-2.0 libbonoboui-2.0 libgnomecanvas-2.0 gtk+-2.0 gdk-pixbuf-2.0 gnome-vfs-2.0
Requires.private: libart-2.0 gconf-2.0 gnome-keyring-1 
Version: 2.24.5
Libs: -L${libdir} -lgnomeui-2 -lSM -lICE
Cflags: -I${includedir}/libgnomeui-2.0

```

**Comment by denisleroy** [comment]
Yes, unfortunately it's not trivial. libgnomeuimm in particular has become a problem as it was never used by many projects.  gcdmaster badly needs a port to Gnome 3. I had started looking at it at some point. but it's a fair amount of work.

**Comment by stefanroellin** [comment]
If you are interested: I have ported gcdmaster some time ago to gtk-3.0 - see https://github.com/stefanroellin/cdrdao

**Comment by denisleroy** [comment]
Oh nice, I had a partial port in a branch but never found the time to finish. Thanks a lot.

**Comment by stefanroellin** [comment]
While the port to gtk-3.0 compiles and works for me, it might not be 100% perfect.
BTW: on sourceforge there is a release of 1.2.4, which I do not see in this repository. I assume this repository is the official version - isn't it? I also see the version 1.2.4 in https://salsa.debian.org/debian/cdrdao.

**Comment by denisleroy** [comment]
Yes this repo replaces the old sourceforge one, although I still have access to it.

**Comment by denisleroy** [comment]
Fixed in 1.2.5
